tests/test_create_model.py

```diff
@@ -516,6 +519,57 @@ def test_create_model_non_annotated():
         create_model('FooModel', foo=(str, ...), bar=123)
 
 
+@pytest.mark.skipif(sys.version_info < (3, 9), reason='typing.Annotated is introduced after python3.9')
```

Comment:
As @Viicos mentioned, you can just pull `Annotated` from `typing_extensions` so that you don't have to skip tests!

Reply:
I'd just import it at the top (`from typing_extensions import Annotated`) and use that everywhere!

Reply:
Then you don't have to duplicate the tests, either!

Reply:
> As @Viicos mentioned, you can just pull `Annotated` from `typing_extensions` so that you don't have to skip tests!

I purposely have added them since I felt it would be best to cover the usage of both typing.Annotated and typing_extensions.Annotated. Would it be fair to assume the usage will always be via typing_extensions.Annotated?

Reply:
> Would it be fair to assume the usage will always be via typing_extensions.Annotated?

People will make use of both `typing` and `typing_extensions`, but I think @sydney-runkle meant that `_typing_extra.is_annotated` is probably already tested eslewhere so we can already be confident it will work with both `typing` and `typing_extensions`

Reply:
Indeed. Feel free to just test the typing extensions version for now. I appreciate the thoroughness, though!